iphone/Maps/Bookmarks/Categories/BMCView/BMCViewController.swift

```diff
@@ -35,6 +35,12 @@ final class BMCViewController: MWMViewController {
   override func viewDidLoad() {
     super.viewDidLoad()
     viewModel = BMCDefaultViewModel()
+    
+    let addButton = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(addTapped))
+    navigationItem.rightBarButtonItem = addButton
+  }
+    @objc func addTapped() {
+    createNewCategory()
```

Comment:
Will this work? (you likely need to add `@objc` annotation to createNewCategory() method.

```suggestion
    navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(createNewCategory))
```

Reply:
Did you test it?